vue/components/ui/molecules/calendar/calendar.vue

```diff
@@ -223,14 +223,18 @@ export const Calendar = {
         },
         selectedYear() {
             return this.valueData?.getFullYear();
+        },
+        valueEpoch() {
+            return Math.round(this.valueData.getTime() / 1000);

```

Comment:
Is it possible to have a non-round value?

Reply:
Yes, I just thought we wanted to represent dates the same way RIPE core does it which doesn't seem to include floating epoch numbers

Reply:
Ok good, just wanted to make sure.